src/pages/PrimeCalculator/RewardDetails/index.tsx

```diff
@@ -1,3 +1,98 @@
-import { Card } from 'components';
+import { Card, Delimiter, Icon, TokenIcon, Tooltip } from 'components';
+import { useTranslation } from 'packages/translations';
+import { Token } from 'types';
 
-export const RewardDetails: React.FC = () => <Card>RewardDetails</Card>;
+interface RewardDetailsProps {
+  token: Token;
+  totalYearlyRewards: string;
+  userYearlyRewards: string;
+  userSuppliedTokens: string;
+  userBorrowedTokens: string;
+  primeSupplyApy: string;
+  primeBorrowApy: string;
+}
+
+export const RewardDetails: React.FC<RewardDetailsProps> = ({
+  primeBorrowApy,
+  primeSupplyApy,
+  token,
+  totalYearlyRewards,
+  userYearlyRewards,
+}: RewardDetailsProps) => {
+  const { t } = useTranslation();
+
+  return (
+    <Card>
+      <h4 className="text-lg font-semibold">{t('primeCalculator.rewardDetails.title')}</h4>
+      <div className="mb-3 mt-6 flex flex-row justify-between">
+        <div className="flex flex-row">
+          <TokenIcon token={token} />
+          <span className="mx-2 text-grey">
+            {t('primeCalculator.rewardDetails.totalYearlyRewards.title')}
+          </span>
+          <Tooltip
+            className="flex flex-col justify-center"
+            title={t('primeCalculator.rewardDetails.totalYearlyRewards.tooltip')}
+          >
+            <Icon name="info" />
+          </Tooltip>
+        </div>
+        <span>{totalYearlyRewards}</span>
+      </div>
+      <div className="mb-6 flex flex-row justify-between">
+        <div className="flex flex-row">
+          <TokenIcon token={token} />
+          <span className="mx-2 text-grey">
+            {t('primeCalculator.rewardDetails.yourYearlyRewards')}
+          </span>
+        </div>
+        <span>{userYearlyRewards}</span>
+      </div>
+
+      <Delimiter />
+
+      <div className="mb-8 mt-6 flex flex-row lg:mb-6">
+        <div className="flex w-40 flex-col">
```

Comment:
Just to confirm, this is the element:

![Untitled](https://github.com/VenusProtocol/venus-protocol-interface/assets/52756735/3b288e09-7736-49c9-8f21-e9bed315faad)

I've double checked the widths in each breakpoint but kept them with a fixed value, let me know what you think

Reply:
Sorry, you're right.
It might be worth raising this to Pavel however, as it does look a bit odd on desktop:
![Uploading Screen Shot 2023-12-19 at 15.25.46.png…]()

Reply:
He confirmed it on Figma